src/components/AddBookForm.jsx

```diff
@@ -26,7 +26,7 @@ function AddBookForm() {
   };
 
   return (
-    <form onSubmit={handleSubmit}>
+    <form className='addForm' onSubmit={handleSubmit}>
```

Comment:
- [ ] Please 🙏🏽 add the title "ADD NEW BOOK" to be similar to the Zeplin design.

| Zeplin | Your design |
|---|---|
| ![image](https://github.com/zamcham/Bookstore/assets/58642949/8be7f0bd-0b51-4d1b-b521-9621bcfa5a31) | ![image](https://github.com/zamcham/Bookstore/assets/58642949/0192fdb6-deee-4855-9307-253a4b597e15) |